eventuate-common-quarkus-jdbc/src/main/java/io/eventuate/common/quarkus/jdbc/DataSourceSelectorConfiguration.java

```diff
@@ -0,0 +1,29 @@
+package io.eventuate.common.quarkus.jdbc;
+
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+import javax.annotation.Priority;
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.inject.Alternative;
+import javax.enterprise.inject.Instance;
+import javax.enterprise.inject.Produces;
+import javax.inject.Named;
+import javax.sql.DataSource;
+
+@ApplicationScoped
+@Alternative
+@Priority(1)
+public class DataSourceSelectorConfiguration {
+
+  @Produces
```

Comment:
As discussed, this is test code. An Eventuate application should not be required to use this.